### ISSUE TITLE
Build Process Video

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#47]
## User story
1. As a user
2. I want to prepare a build process video
3. So that I can demonstrate the steps to build the project

## Acceptance criteria
* The video covers the complete build process
* Consider the common steps described in the "Build process video" homework section
* The video is clear, concise, and easy to follow

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
